ui-v2/app/instance-initializers/selection.js

```diff
@@ -5,7 +5,8 @@ const isSelecting = function(win = window) {
   const selection = win.getSelection();
   let selecting = false;
   try {
-    selecting = 'isCollapsed' in selection && !selection.isCollapsed;
+    selecting =
+      'isCollapsed' in selection && !selection.isCollapsed && selection.toString().length > 1;
```

Comment:
is there any risk that selection would be `null`? If so, probably better to add a guard clause

Reply:
I'm 'pretty' sure getSelection() always returns a selection object here:

https://developer.mozilla.org/en-US/docs/Web/API/Window/getSelection

We also have across the board browser support for that

It would also just throw an error to be caught if anything 'goes wrong' here (which would mean selecting = false).

What do you think, happy to add if you think there's a risk? Lemme know.

Reply:
Hey @gregone 

Just to let you know I'll probably merge this this afternoon if I don't hear anymore.

Thanks!

John